system/jlib/jptree.cpp

```diff
@@ -8936,19 +8936,7 @@ static std::tuple<std::string, IPropertyTree *, IPropertyTree *> doLoadConfigura
 
     //For legacy (and other weird cases) ensure there is a global section
     if (!newGlobalConfig)
-    {
         newGlobalConfig.setown(createPTree("global"));
-#ifndef _CONTAINERIZED
```

Comment:
Why remove this section?  Does it remove backward compatibility?

Reply:
It doesn't remove backward compatibility.  It has been removed because there is no longer necessary....

Previously, the configuration the bare-metal global configuration did not exist in the configuration file - it was dynamically generated at run-time.  As of [8.6.x ](https://track.hpccsystems.com/browse/HPCC-27688) the global section was created for some components.  Because the global section "exists" now, this section will never be executed and the "cost" section will not be moved into the global config.  For this PR, the cost section is being generated within the component's configuration by the .xsl.  It is no longer necessary to copy it in because it's already there.

Reply:
this is a bare-metal issue only and assuming the xsl's that generate the per component config files each time a component starts (e.g. thor.xml) are in sync with the build, then afaics, it is correct to remove this code.

Agree that It is no longer necessary (this code) to 'move' the "cost" that was generated outside of the global section by the xslt's to inside it.